ui/raidboss/data/05-shb/eureka/bozjan_southern_front.txt

```diff
@@ -365,4 +365,3 @@ hideall "--sync--"
 61126.0 "Frigid/Fervid Pulse"
 61128.4 "Swooping Frenzy"
 61131.2 "Frigid/Fervid Pulse"
-
```

Comment:
Unintentional removal of the blank line at EOF?

Reply:
> Unintentional removal of the blank line at EOF?

There were 2, so I removed one.  For whatever reason, I don't think GitHub (web) shows the final blank line at EOF, even though it is present in the file (at least, that seems to be the case looking at a few other files).  But the local version I have definitely has an empty 368 (and no longer a 369)...